<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateLiveUI</class>
 <widget class="QDialog" name="CreateLiveUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ButelLive</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(35, 35, 37,255);
border-image: url(:/images/xxx_bg.png);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="TitleBar" name="m_pTitleLbl" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(44, 44, 44);</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>35</y>
        <width>61</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CurChannel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>61</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>LiveName</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_pLdtProgramName">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>90</y>
        <width>290</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 31, 32);
color: rgb(255, 255, 255);
border:0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="m_pLbCurChannel">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>35</y>
        <width>290</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="m_pLbPicture">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>150</y>
        <width>180</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="m_pBtnUpload">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>150</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/btn_upload_bg.png);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_pBtnGoLive">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>290</y>
        <width>350</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/btn_golive_bg.png);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>GoLive</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="m_pProgressBar">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>250</y>
        <width>180</width>
        <height>6</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>6</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>6</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background:#dddfe5;
border-radius:3px;
text-align:center;
}
QProgressBar::chunk {
background-color:#1eb5dc;
border-radius:3px;</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header location="global">titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
